modules/apps/frontend-js/frontend-js-components-web/src/main/java/java/com/liferay/frontend/js/components/web/internal/servlet/taglib/WalkthroughBottomJSPDynamicInclude.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.components.web.internal.servlet.taglib;
+
+import com.liferay.frontend.js.loader.modules.extender.npm.NPMResolver;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.servlet.taglib.DynamicInclude;
+import com.liferay.portal.kernel.servlet.taglib.aui.ScriptData;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Matuzalem Teles
+ * @author Diego Nascimento
+ */
+@Component(service = DynamicInclude.class)
+public class WalkthroughBottomJSPDynamicInclude implements DynamicInclude {
+
+	@Override
+	public void include(
+			HttpServletRequest httpServletRequest,
+			HttpServletResponse httpServletResponse, String key)
+		throws IOException {
+
+		ScriptData scriptData = new ScriptData();
+
+		String resolvedModuleName = _npmResolver.resolveModuleName(
+			"frontend-js-components-web/walkthrough/WalkthroughRender");
+
+		System.out.println(resolvedModuleName);
+
+		JSONObject propsJSONObject = JSONUtil.put("sampleProp", Boolean.TRUE);
+
+		scriptData.append(
```

Comment:
I'm not sure but probably yes. This part of DynamicIncluide we will ignore because we need to do another one based on the ticket requirements https://issues.liferay.com/browse/LPS-150904. We made this implementation just to be able to test the logic on the React side running on all pages.

Reply:
yep that makes sense Matu